exporter/loadbalancingexporter/README.md

```diff
@@ -66,13 +66,13 @@ Refer to [config.yaml](./testdata/config.yaml) for detailed examples on using th
   * `service` Kubernetes service to resolve, e.g. `lb-svc.lb-ns`. If no namespace is specified, an attempt will be made to infer the namespace for this collector, and if this fails it will fall back to the `default` namespace.
   * `ports` port to be used for exporting the traces to the addresses resolved from `service`. If `ports` is not specified, the default port 4317 is used. When multiple ports are specified, two backends are added to the load balancer as if they were at different pods.
   * `timeout` resolver timeout in go-Duration format, e.g. `5s`, `1d`, `30m`. If not specified, `1s` will be used.
-* The `awsCloudMap` node accepts the following properties:
+* The `aws_cloud_map` node accepts the following properties:
   * `namespace` The CloudMap namespace where the service is register, e.g. `cloudmap`. If no `namespace` is specified, this will fail to start the Load Balancer exporter.
-  * `serviceName` The name of the service that you specified when you registered the instance, e.g. `otelcollectors`.  If no `serviceName` is specified, this will fail to start the Load Balancer exporter.
+  * `service_name` The name of the service that you specified when you registered the instance, e.g. `otelcollectors`.  If no `serviceName` is specified, this will fail to start the Load Balancer exporter.
```

Comment:
```suggestion
  * `service_name` The name of the service that you specified when you registered the instance, e.g. `otelcollectors`.  If no `service_name` is specified, this will fail to start the Load Balancer exporter.
```